examples/upload_files.py

```diff
@@ -22,25 +22,36 @@ async def get_files_to_upload(queue, path):
   """Finds which files will be uploaded to VirusTotal."""
   if os.path.isfile(path):
     await queue.put(path)
-    return
+    return 1
 
+  n_files = 0
   with os.scandir(path) as it:
     for entry in it:
         if not entry.name.startswith('.') and entry.is_file():
             await queue.put(entry.path)
+            n_files += 1
+  return n_files
 
 
 async def upload_hashes(queue, apikey):
   """Uploads selected files to VirusTotal."""
   async with vt.Client(apikey) as client:
     while not queue.empty():
       file_path = await queue.get()
-      await client.scan_file_async(file=file_path)
-      print(f'File {file_path} uploaded.')
-      queue.task_done()
+      with open(file_path) as f:
+        analysis = await client.scan_file_async(file=f)
+        print(f'File {file_path} uploaded.')
+        queue.task_done()
+  return (analysis, file_path)
```

Comment:
You're returning the last file path instead of all of them.

Reply:
fixed at [87f44e4](https://github.com/VirusTotal/vt-py/pull/94/commits/87f44e48d50c822abd0eba6ddb5c788e85bc32ef)